### PR TITLE
tests: e2e_regression add edenfast suite

### PR DESCRIPTION
**Pull request by pro-wh**
fast sync 500 blocks

### CODE REVIEW

**Comment by pro-wh** [comment]
```diff
diff --git a/tmp/nexus-e2e-expected-edenfast/db__debonding_delegations.csv b/home/wh0/work/ekiden/explorer/indexer/tests/e2e_regression/edenfast/actual/db__debonding_delegations.csv
index 65db9985..c6a99abd 100644
--- a/tmp/nexus-e2e-expected-edenfast/db__debonding_delegations.csv
+++ b/home/wh0/work/ekiden/explorer/indexer/tests/e2e_regression/edenfast/actual/db__debonding_delegations.csv
@@ -272,6 +272,7 @@ debond_end,delegator,delegatee,shares
 0,oasis1qq0c0mrvpqteprgdznuevm3s47qwat6qcyfewrfz,oasis1qpavd66xsezz8s4wjw2fyycxw8jm2nlpnuejlg2g,2857102872414
 0,oasis1qq0kf6pqaatc426p4ps6ppgxplw47xu7qc3exaxj,oasis1qq0xmq7r0z9sdv02t5j9zs7en3n6574gtg8v9fyt,1517696826457
 0,oasis1qq0kyhsmc7f8glahaq473r0v5r8uhwwh0vnjykef,oasis1qq3xrq0urs8qcffhvmhfhz4p0mu7ewc8rscnlwxe,855408493311805
+0,oasis1qq0rdl4jpaat3wxj9jznmvn0e8y8wl8geg35h0a5,oasis1qq3xrq0urs8qcffhvmhfhz4p0mu7ewc8rscnlwxe,1721640707086
 0,oasis1qq0sfa3kyesue998lyt6z6zw7yqsph0cyqzmp88z,oasis1qp4tj3u9qkcgjqrrjvwljrqcyx3g5ygjqgtm37t3,5125515449132
 0,oasis1qq0sfa3kyesue998lyt6z6zw7yqsph0cyqzmp88z,oasis1qqf6wmc0ax3mykd028ltgtqr49h3qffcm50gwag3,1423600341598
 0,oasis1qq0w235axax2h9h3andhqt43u288mkyunydtetmz,oasis1qr5dfqd9qw4k9829ajt2xa873gwuwjdmusc0smdh,32000000000
@@ -605,6 +606,7 @@ debond_end,delegator,delegatee,shares
 0,oasis1qral2lrwsex5c00e62dc0un0hczk0axqcynp6cxt,oasis1qz2rxs50p4kn537tcupjhg7ad3zcm9s7uuyrrd9e,35425384199
 0,oasis1qrapc9zfkka00m4w5arjpc8jtp3u6e67tge6j0ss,oasis1qq0xmq7r0z9sdv02t5j9zs7en3n6574gtg8v9fyt,5024368792645277
 0,oasis1qraq35mea6hwd306qpweeymrahxe242y8q9njp28,oasis1qq3xrq0urs8qcffhvmhfhz4p0mu7ewc8rscnlwxe,997231240365
+0,oasis1qrc22lp0ehxdu7wrrchnj8z4e0xwjw9ccu25yllw,oasis1qpqz2ut6a6prfcjm64xnpnjhsnqny6jqfyav829v,175493132658726
 0,oasis1qrc2zn4caahhxcnjyw7jrayc0y0hz8xhgg2r52nx,oasis1qqekv2ymgzmd8j2s2u7g0hhc7e77e654kvwqtjwm,5941109585275
 0,oasis1qrcaaaxffujqml44xhf625edk7pday7a8gfrzll7,oasis1qq0xmq7r0z9sdv02t5j9zs7en3n6574gtg8v9fyt,16095080517133
 0,oasis1qrchfqz6jt527samdegetpd3ugn3e6ddwq4fcyus,oasis1qp60saapdcrhe5zp3c3zk52r4dcfkr2uyuc5qjxp,1574588353344
@@ -1114,8 +1116,6 @@ debond_end,delegator,delegatee,shares
 0,oasis1qzztta40jwtn9rymt4uktgdjmh3jmkh4kq5rnghx,oasis1qq3xrq0urs8qcffhvmhfhz4p0mu7ewc8rscnlwxe,23651999999998
 0,oasis1qzzwzuf4y6cjjeqs6xrrtj4h9krnxzfufcxlgpgw,oasis1qpqz2ut6a6prfcjm64xnpnjhsnqny6jqfyav829v,5011355870
 28353,oasis1qpaguaw93zjpue5hqrzk33ucnslpvct52yqdru34,oasis1qq0xmq7r0z9sdv02t5j9zs7en3n6574gtg8v9fyt,1148214741561
-28353,oasis1qq0rdl4jpaat3wxj9jznmvn0e8y8wl8geg35h0a5,oasis1qq3xrq0urs8qcffhvmhfhz4p0mu7ewc8rscnlwxe,1721640707086
-28353,oasis1qrc22lp0ehxdu7wrrchnj8z4e0xwjw9ccu25yllw,oasis1qpqz2ut6a6prfcjm64xnpnjhsnqny6jqfyav829v,175493132658726
 28353,oasis1qrmljq4v27nv5g3a5xqgnggyk72vyrvxfc78w56c,oasis1qrgmg0t3zxn4lgary6w227dv6c5dl55rfvh6p6k2,1309013487429
 28353,oasis1qrzurlegdkaedftu8k6w8n27dvvytfprjv055ejc,oasis1qq3xrq0urs8qcffhvmhfhz4p0mu7ewc8rscnlwxe,42185242132980
 28353,oasis1qzkjfk98607jlx6mh7r0ejxwkahup3566qylnvnh,oasis1qpqz2ut6a6prfcjm64xnpnjhsnqny6jqfyav829v,1164773308499864
```

File: .github/workflows/ci-test.yaml [hunk 1]
```diff
@@ -162,6 +162,37 @@ jobs:
             tests/e2e_regression/eden/actual
         if: failure()  # but not if success or job is manually cancelled
 
+  test-e2e-regression-edenfast:
```
[the remark this answers is not in the document]
Comment (mitjat) [reply .github/workflows/ci-test.yaml:165]: But don't block this PR on it; if you're not feeling it, it's fine to leave a TODO and we'll do it "eventually" :), e.g. when we add a fourth suite.

File: tests/e2e_regression/edenfast/e2e_config_1.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Config for phase 1 of 2 of e2e regression tests.
```
Comment (mitjat) [inline tests/e2e_regression/edenfast/e2e_config_1.yml:1]: Can you add a paragraph describing the goal of this suite. Basically its relation to `eden`.

**Review by mitjat — APPROVED** [review]
Nice. When I suggested/requested testing fast_sync, I thought you'd just do a quick&dirty offline tweak to the tests ... but we got something better instead :). Thank you!

I have just one substantial comment about comparing `edenfast` vs `eden`.

File: .github/workflows/ci-test.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (pro-wh) [reply .github/workflows/ci-test.yaml:167]: nested loops in yaml, going for it

File: tests/e2e_regression/edenfast/test_cases.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (pro-wh) [reply tests/e2e_regression/edenfast/test_cases.sh:1]: ya they're meant to be the same. shoot, is the copy out of date

leme try with the `expected` symlink

[the remark this answers is not in the document]
Comment (pro-wh) [reply tests/e2e_regression/edenfast/test_cases.sh]: works. adding symlink check to accept.sh

File: tests/e2e_regression/edenfast/e2e_config_1.yml (hunk 2 above)
Comment (pro-wh) [reply tests/e2e_regression/edenfast/e2e_config_1.yml:1]: adding

**Comment by pro-wh** [comment]
ok why are the mock ones running

**Pull request merged**